src/pizzeria/Pizza.java

```diff
@@ -0,0 +1,80 @@
+package pizzeria;
+
+public class Pizza {
+    private static final int MAX_ALLOWED_INGREDIENTS_SIZE = 7;
+
+    private String name;
+    private PizzaType pizzaType;
+    private Ingredients[] ingredients;
+    private double price;
+    private int ingredientsCount;
+    private Object Ingredients;
+
+    public Pizza(String name) {
+        this.name = name;
+        this.ingredients = new Ingredients[MAX_ALLOWED_INGREDIENTS_SIZE];
+        this.pizzaType = PizzaType.getRegularType();
+    }
+
+    public Pizza(String name, PizzaType pizzaType) {
+        this.name = name;
+        this.ingredients = new Ingredients[MAX_ALLOWED_INGREDIENTS_SIZE];
+        this.pizzaType = pizzaType;
+    }
+
+    public Pizza(String name, PizzaType pizzaType, Ingredients[] ingredients) {
+        this.name = name;
+        this.pizzaType = pizzaType;
+        if (ingredients.length > MAX_ALLOWED_INGREDIENTS_SIZE) {
+            System.out.println("Ingredients more than allowed 7." + MAX_ALLOWED_INGREDIENTS_SIZE);
+            this.ingredients = new Ingredients[MAX_ALLOWED_INGREDIENTS_SIZE];
```

Comment:
If ingredients length is > MAX_ALLOWED_INGREDIENTS_SIZE, then your this.ingredients will be array of null values, pls check how you can fix it